smartmin/views.py

```diff
@@ -117,13 +117,8 @@ def has_object_permission(self, getter_name):
         """
         Checks for object level permission for an arbitrary getter
         """
-        obj = None
         obj_getter = getattr(self, getter_name, None)
 
-        # get object requires pk
-        if getter_name == "get_object" and 'pk' not in self.kwargs:
-            return False
```

Comment:
There's no reason to validate this here before just calling the getter. We don't know what kwargs it depends on. (or slugs)